### PR TITLE
Updated example to use NVMe storage

### PR DESCRIPTION
**Pull request by chris-milsted**

### CODE REVIEW

File: docs/install/amazon-web-services-eks.md [hunk 1]
```diff
@@ -59,34 +60,89 @@ iam:
...
+    availabilityZones:
+      - eu-west-2c
+    minSize: 1
+    maxSize: 3
+    desiredCapacity: 1
+    instanceType: i3en.xlarge
+    amiFamily: Ubuntu2004
+    ssh:
+      allow: true
+      publicKeyName: <key-name>
```
Comment (Arau) [inline docs/install/amazon-web-services-eks.md:123]: I would make a comment at the end of this yaml to suggest what fields need to be modified. In this case I believe the only one is the publicKeyName.

**Review by Arau — APPROVED** [review]
Thank you for the changes!

**Comment by chris-milsted** [comment]
@cvlc - hoping I cleared these now for review.

File: docs/install/amazon-web-services-eks.md (hunk 1 above)
Comment (chris-milsted) [reply docs/install/amazon-web-services-eks.md:123]: Added!

**Review by cvlc — DISMISSED** [review]
LGTM

**Review by Arau — APPROVED** [review]

**Comment by chris-milsted** [comment]
@cvlc - trying again

**Review by Arau — APPROVED** [review]

**Pull request merged**